spec/journal_title_spec.rb

```diff
@@ -445,7 +448,7 @@
   end
 
   context "the journal" do
-    it_behaves_like "great results for journal/newspaper", "The journal" do
+    it_behaves_like "great results for journal/newspaper", "The journal", :fixme => true do
```

Comment:
I no longer remember - did we give up on "The journal"?  Or did you just forget to remove the fixme?

Reply:
Same as above.